### ISSUE TITLE
Miscellaneous

### ISSUE THREAD

**Issue by gjain-7** [Web-Team-IITI-Gymkhana/tpc-backend#69]
- [ ] Change `year` column in `SeasonModel` from String to Int with constraints (between 2000 and 2100) and elsewhere.
- [ ] Use `JobStatusModel` to note transitions 
- [x] Add a Logs Table

**Comment by NAGAMALLYSRUJAN2329** [comment]
Should I also change the year in the ProgramModel?

**Comment by gjain-7** [comment]
> Should I also change the year in the ProgramModel?

Yeah sure you can pick up this issue once the #56  is closed, which should be the case by today's EOD. Additionally, you may also need to change the DTOs using that field.

**Comment by gjain-7** [comment]
> Should I also change the year in the ProgramModel?

You can start working on this one
